src/common/monitors.cpp

```diff
@@ -1,84 +1,64 @@
 #include "pch.h"
 
-#include "common.h"
 #include "monitors.h"
 
+#include "common.h"
+
 bool operator==(const ScreenSize& lhs, const ScreenSize& rhs)
 {
     auto lhs_tuple = std::make_tuple(lhs.rect.left, lhs.rect.right, lhs.rect.top, lhs.rect.bottom);
     auto rhs_tuple = std::make_tuple(rhs.rect.left, rhs.rect.right, rhs.rect.top, rhs.rect.bottom);
     return lhs_tuple == rhs_tuple;
 }
 
-static BOOL CALLBACK get_displays_enum_cb(HMONITOR monitor, HDC hdc, LPRECT rect, LPARAM data)
+static BOOL CALLBACK GetDisplaysEnumCb(HMONITOR monitor, HDC hdc, LPRECT rect, LPARAM data)
 {
-    MONITORINFOEX monitor_info;
-    monitor_info.cbSize = sizeof(MONITORINFOEX);
-    GetMonitorInfo(monitor, &monitor_info);
-    reinterpret_cast<std::vector<MonitorInfo>*>(data)->emplace_back(monitor, monitor_info.rcWork);
+    MONITORINFOEX monitorInfo;
+    monitorInfo.cbSize = sizeof(MONITORINFOEX);
+    if (GetMonitorInfo(monitor, &monitorInfo))
+    {
+        reinterpret_cast<std::vector<MonitorInfo>*>(data)->emplace_back(monitor, monitorInfo.rcWork);
+    }
     return true;
 };
 
-static BOOL CALLBACK get_displays_enum_cb_with_toolbar(HMONITOR monitor, HDC hdc, LPRECT rect, LPARAM data)
+static BOOL CALLBACK GetDisplaysEnumCbWithToolbar(HMONITOR monitor, HDC hdc, LPRECT rect, LPARAM data)
 {
-    MONITORINFOEX monitor_info;
-    monitor_info.cbSize = sizeof(MONITORINFOEX);
-    GetMonitorInfo(monitor, &monitor_info);
-    reinterpret_cast<std::vector<MonitorInfo>*>(data)->emplace_back(monitor, monitor_info.rcMonitor);
+    MONITORINFOEX monitorInfo;
+    monitorInfo.cbSize = sizeof(MONITORINFOEX);
+    if (GetMonitorInfo(monitor, &monitorInfo))
+    {
+        reinterpret_cast<std::vector<MonitorInfo>*>(data)->emplace_back(monitor, monitorInfo.rcMonitor);
+    }
     return true;
 };
 
 std::vector<MonitorInfo> MonitorInfo::GetMonitors(bool include_toolbar)
```

Comment:
`include_toolbar` is not a proper name, the difference is not about the toolbar, but about full screen vs working area, the toolbar is just one of the possible things that can effect the available working area, but it's not the only one, so the name is definitely misleading. Same as above, this can be fixed in a different PR.